libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -108,6 +109,13 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FRAME("AUTOENABLE", 10, AC_Fence, _auto_enabled, static_cast<uint8_t>(AutoEnable::ALWAYS_DISABLED), AP_PARAM_FRAME_PLANE),
 
+    // @Param: ALT_FRAME
+    // @DisplayName: Fence Altitude Frame
+    // @Description: Frame for min and max altitude, default is 1; MSL = 0, REL/HOME = 1, AGL = 2
+    // @Range: 0 1 2
```

Comment:
Instead of range, I think you should use @values here, and add a text to each value.